components/http_foundation/session_configuration.rst

```diff
@@ -87,6 +88,35 @@ Example usage::
     $sessionStorage = new NativeSessionStorage(array(), new PdoSessionHandler($pdo));
     $session = new Session($sessionStorage);
 
+Migrating Between Save Handlers
+-------------------------------
+
+The :class:`Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\MigratingSessionHandler`
+can be used to migrate between old and new save handlers without losing session data.
+
+It can be used to support the following workflow:
+
+* Switch to the migrating handler, with your new handler as the write-only one. The old handler behaves as usual and sessions get written to the new one.
+* After your session gc period, verify the data in the new handler
+* Update the migrating handler to use the old handler as the write-only one, so the sessions will now be read from the new handler. This step allows easier rollbacks.
+* After verifying everything, switch from the migrating handler to the new handler
```

Comment:
Missing a full stop at the end

` * After verifying everything, switch from the migrating handler to the new handler.`